<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CredentialsDialog</class>
 <widget class="QDialog" name="CredentialsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="credentialsTable">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>361</width>
     <height>261</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>URL</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Login</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Password</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="filterInput">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="filterButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Фильтр</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
